platform/viewer/src/utils/cornerstoneWADOImageLoader.js

```diff
@@ -0,0 +1,57 @@
+import OHIF from '@ohif/core';
```

Comment:
Maybe we should move all of these `getSomeExternalDependency` methods to a folder so they're easier to locate? I think we'll eventually want to make this split possible/easier with dependency injection instead of using a utility like this to isolate the import?